script/DeployScript.s.sol

```diff
@@ -30,7 +30,7 @@ contract DeployScript is Script {
         // wrap in ABI to support easier calls
         AllowList wrappedPerms = AllowList(address(permsProxy));
 
-        tokenImplementation = new USTB(admin, wrappedPerms);
+        tokenImplementation = new USTB( wrappedPerms);
```

Comment:
```suggestion
        tokenImplementation = new USTB(wrappedPerms);
```

Reply:
just run forge fmt. we should enfroce this on the mon repo 👍